app/admin/founder.rb

```diff
@@ -1,13 +1,9 @@
 ActiveAdmin.register Founder do
   actions :index, :show
 
-  permit_params :name, :remote_avatar_url, :avatar, :startup_id, :about, :communication_address, :phone,
-    :college_id, :twitter_url, :linkedin_url, :personal_website_url, :blog_url, :angel_co_url, :github_url,
-    :behance_url, :skype_id, :excluded_from_leaderboard, roles: [], tag_list: []
+  permit_params :name, :remote_avatar_url, :avatar, :startup_id, :about, :college_id, :excluded_from_leaderboard, roles: [], tag_list: []
```

Comment:
Unable to find usages of `:remote_avatar_url`
```suggestion
  permit_params :name, :avatar, :startup_id, :about, :college_id, :excluded_from_leaderboard, roles: [], tag_list: []
```